public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -1384,7 +1380,7 @@ a(href="#toc") Back to top
 
 .s-rule.do
   :marked
-    **Do** collect single-use classes and hiding their gory details inside `CoreModule`. A simplified root `AppModule` imports `CoreModule` in its capacity as orchestrator of the application as a whole.
+    **Do** collect single-use classes and hide their gory details inside `CoreModule`. A simplified root `AppModule` imports `CoreModule` in its capacity as orchestrator of the application as a whole.
```

Comment:
Better?:
>**Consider** collecting numerous, auxiliary, single-use classes inside a core module
to simplify the apparent structure of a feature module. 
>
>**Consider** calling the application-wide core module, `CoreModule` .
Importing `CoreModule` into the root  `AppModule`  reduces its complexity and emphasizes its role as orchestrator of the application as a whole.